pyrightconfig.json

```diff
@@ -0,0 +1,3 @@
+{
+  "ignore": ["./starknet_in_rust/", "./blockifier/"]
+}
```

Comment:
Let's add a newline here.

Reply:
I added this file in [kb/1906-add-test-rust-command](https://github.com/software-mansion/protostar/tree/kb/1906-add-test-rust-command) with a new line